controllers/managedclusterForCGU_controller.go

```diff
@@ -123,11 +123,6 @@ func (r *ManagedClusterForCguReconciler) Reconcile(ctx context.Context, req ctrl
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-		if len(policies) == 0 {
-			// likely no policies were created, so no child policies found
-			r.Log.Info("WARN: No child policies found for cluster", "Name", managedCluster.Name)
```

Comment:
I think we can keep the WARNING log for debug purpose

Reply:
done